### ISSUE TITLE
Please provide the wheel file for ParallelSampler, if possible.

### ISSUE THREAD

**Issue by intelsam** [facebookresearch/shaDow_GNN#6]
Hello, can you please provide the wheel file for ParallelSampler?
The reason I ask is that there are some issues on my machine with building the ParallelSampler package.
Additionally, it makes life easier if someone wants to run the repo in Collab for faster experiments and prototyping.

By the way, building the wheel does not require any extra work as it's automatically generated when you build it.

PS: I know that the wheel file will be dependent on matching the other package versions. However, I can make sure to install matching versions.

**Comment by ZimpleX** [comment]
Hello, thanks for keep looking into this issue. 

It seems that wheel is deleted after pip install is completed. So I tried to manually generate the files. 

In the attached zip, there are two wheels:
* ParallelSampler corresponds to the package in `para_graph_sampler/graph_engine/backend`
* para_graph_sampler corresponds to the overall package in `para_graph_sampler`

I haven't gone through the process of package matching of wheels. So I'm not sure if these info are sufficient for you. Please let me know if you need any other inputs. 

[Archive.zip](https://github.com/facebookresearch/shaDow_GNN/files/10212645/Archive.zip)

**Comment by intelsam** [comment]
Thanks a ton @ZimpleX . Really really appreciate your effort in providing the wheel files.
Unfortunately, the wheel files didn't help much but I have solved the issue I was facing.

I think Ubuntu 16.04 was the primary issue or some config in the OS. I created a docker container with this [image](https://hub.docker.com/layers/pytorch/pytorch/1.11.0-cuda11.3-cudnn8-devel/images/sha256-9bfcfa72b6b244c1fbfa24864eec97fb29cfafc065999e9a9ba913fa1e690a02?context=explore) and everything works fine now.
Thanks a lot again for all your help and effort.
